README.md

```diff
@@ -174,6 +174,19 @@ curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyy
 curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyyja" localhost:3001/v0/profile
 ```
 
+## Development Setup ( Docker )
```

Comment:
It is both for developers who don't want to deal with setting it up, but also to ease both the testing and development setup overhead. the compose file create all the necessary infrastructure with a single command ( Postgres, Redis, Specific version of Ruby ) so it reduces the amount of time someone would need to spend preparing to work on the project. 

Then from a operations perspective we can also turn more of our build servers into general purpose builders.